src/ralph/trade_marks/admin.py

```diff
@@ -22,6 +22,7 @@ class TradeMarksLinkedView(RalphDetailViewAdmin):
 
     class TradeMarksLinkedInline(RalphTabularInline):
         model = TradeMarksLinkedDomains
+        verbose_name_plural = "TradeMarksLinkedDomains"
```

Comment:
1. ``verbose_name*`` is for humans 
2. add support for translation

Reply:
is it really necessary in inline (why not on model)?